docs/known_issues.md

```diff
@@ -23,6 +23,10 @@ metadata:
 type: Opaque
 ```
 
+## Storage class
+
+Velero Plugin for vSphere v1.1 requires a storage class to restore a PVC. This is because a PVC will be provisioned dynamically at the restore time. If a storage class is not available, restore will fail. For statically provisioned volumes, user can specify `com.vmware.cnsdp.emptystorageclass` as the old storage class name to map to a new existing storage class name at restore time.
```

Comment:
v1.1 -> v1.1.0 ?

Reply:
I put v1.1 there on purpose so that we don't have to update this when we do a patch release.  Maybe I should say v1.1.0 and higher instead.

Reply:
Done